### ISSUE TITLE
doc

### ISSUE THREAD

**Issue by alex-nexus** [jmansor/ipsy-music#6]
Just curious, did you code the api doc's json file?

**Comment by jmansor** [comment]
No, the JSON for the documentation is automatically generated by a gem called rspec_api_documentation. It is built on top of rspec request specs and let's you add some metadata to tests that will be used to generate the documentation. It outputs the JSON files, which will later be user by a Viewer (in this case Apitome), which is the one that actually formats the documentation visually.
